demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -182,11 +185,13 @@ const BACKGROUND_BLUR_ASSET_SPEC = (BACKGROUND_BLUR_ASSET_GROUP || BACKGROUND_BL
   revisionID: BACKGROUND_BLUR_REVISION_ID,
 }
 
-type VideoFilterName = 'Emojify' | 'NoOp' | 'Segmentation' | 'Resize (9/16)' | 'CircularCut' |
+type VideoFilterName = 'Emojify' | 'NoOp' | 'Resize (9/16)' | 'CircularCut' |
  'Background Blur 10% CPU' | 'Background Blur 20% CPU' | 'Background Blur 30% CPU' | 
  'Background Blur 40% CPU' | 'Background Replacement' | 'None' | 'Background Blur 2.0 - Low' |
  'Background Blur 2.0 - Medium' | 'Background Blur 2.0 - High' | 'Background Replacement 2.0 - (Beach)' |
- 'Background Replacement 2.0 - (Blue)' | 'Background Replacement 2.0 - (Default)';
+ 'Background Replacement 2.0 - (Blue)' | 'Background Replacement 2.0 - (Default)' |
+  'MediaPipeBodySegmentation - (Beach)' | 'MediaPipeBodySegmentation - (Blue)' |
+  'MediaPipeBodySegmentation - (Blur)';
```

Comment:
Maybe we can just have one background for both 2.0 and media pipe :). Also i think we can remove my beloved emojify filter.

On a side note, should we fully deprecate v1? I can ask internally but it seems odd to still have two options 2 years after releasing v2.

Reply:
The react library and others still use v1 I think :D